frontend/src/components/Pages/FrontPage/MissionOverview/StopMissionDialog.tsx

```diff
@@ -0,0 +1,100 @@
+import { Button, Dialog, Typography, Icon } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+import { TranslateText } from 'components/Contexts/LanguageContext'
+import { Icons } from 'utils/icons'
+import { useState, useEffect } from 'react'
+import { tokens } from '@equinor/eds-tokens'
+import { Mission } from 'models/Mission'
+import { useMissionControlContext } from 'components/Contexts/MissionControlContext'
+
+const StyledDisplayButtons = styled.div`
+    display: flex;
+    width: 410px;
+    flex-direction: columns;
+    justify-content: flex-end;
+    gap: 0.5rem;
+`
+
+const StyledDialog = styled(Dialog)`
+    display: grid;
+    width: 450px;
+`
+
+const StyledText = styled.div`
+    display: grid;
+    gird-template-rows: auto, auto;
+    gap: 1rem;
+`
+
+interface MissionProps {
+    mission: Mission
+}
+
+export enum ControlButton {
+    Pause,
+    Stop,
+    Resume,
+}
+
+export const StopMissionDialog = ({ mission }: MissionProps): JSX.Element => {
+    const [isStopMissionDialogOpen, setIsStopMissionDialogOpen] = useState<boolean>(false)
+    const [missionId, setMissionId] = useState<string>()
+    const { handleClick } = useMissionControlContext()
+
+    const handleDialog = () => {
+        setIsStopMissionDialogOpen(true)
+        setMissionId(mission.id)
+    }
+
+    useEffect(() => {
+        if (missionId !== mission.id) setIsStopMissionDialogOpen(false)
+    }, [mission.id])
+
+    return (
+        <>
+            <Button variant="ghost_icon" onClick={() => handleDialog()}>
```

Comment:
```suggestion
            <Button variant="ghost_icon" onClick={handleDialog}>
```
If the function "handleDialog" doesn't take any arguments this is possible.
Would also consider renaming to "openDialog"